server/models/UserInterests.model.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  AllowNull,
+  Column,
+  CreatedAt,
+  ForeignKey,
+  Model,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript';
+
+interface User_InterestsAttributes {
+  user_id: number;
+  interest_name: string;
```

Comment:
`interest_name` should be `interest_id` because this will map to an interest id on the interest table